samples/storage_csm_powerflex_v220.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: storage.dell.com/v1
```

Comment:
This file should be removed if we decide to support n-2 versions of driver

Reply:
removed it